Samples/Litium App demo/Resources/AppConfig/AppMetadata.json

```
@@ -0,0 +1,27 @@
+{
+  "AppMetadata": {
+    "Version": "1.0.0",
+    "Id": "AppDemo",
+    "DisplayName": "App Demo",
+    "RequestedPermissions": [
+      "Function/SystemSettings"
```

Should we change to?
```suggestion
      "Function/Connect/Payments"
```

---

It is Litium App demo, not payment connector demo.

---

I know, but having System permission here seems not practical. In fact it can be a bad practice when giving an app a system permission.